README.md

```diff
@@ -217,13 +217,15 @@ ios/tmp.xcconfig
 4. go to project settings
 5. apply config to your configurations
    ![img](./readme-pics/3.ios_apply_config.png)
-6. create new build phase for the scheme which will generate "tmp.xcconfig" before each build exposing values to Build Settings and Info.plist (this snippet has to be placed after "cp ... ${PROJECT_DIR}/../.env" if [approach explained below](#ios-multi-scheme) is used)
+6. Go to _Edit scheme..._ -> _Build_ -> _Pre-actions_, click _+_ and select _New Run Script Action_. Paste below code which will generate "tmp.xcconfig" before each build exposing values to Build Settings and Info.plist. Make sure to select your target under _Provide build settings from_, so `$SRCROOT` environment variables is available to the script. (Note that this snippet has to be placed after "cp ... \${PROJECT_DIR}/../.env" if [approach explained below](#ios-multi-scheme) is used).
```

Comment:
Looks correct to me.

Reply:
I was having trouble with the previous build phase run script.
Then I added it to pre-actions and it worked fine. But since I was having multiple targets, It's a bit annoying when it comes to CI/CD.
So what I did was, I moved the run script to the top right before **Compile Sources** as **RN Config** in the build phase as follows and it worked for me.

![image](https://user-images.githubusercontent.com/18027269/98036324-d4fc2780-1e3f-11eb-8d4d-d845438b180c.png)